crates/sui-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap

```diff
@@ -90,6 +90,8 @@ validator_configs:
       shared_object_disabled: false
       user_transaction_disabled: false
     certificate-deny-config: {}
+    state-debug-dump-config:
+      dump-file-directory: /var/folders/3m/hnrdf4892mj0df4tqf55pm_m0000gn/T/
```

Comment:
this of course won't pass tests - i like @mwtian's suggestion of defaulting it to empty and then returning a tempdir if it is unset.

Reply:
Yup, fixing that right now.